nemo/core/__init__.py

```diff
@@ -16,6 +16,7 @@
 # limitations under the License.
 
 from nemo.core.callbacks import *
+from nemo.core.nemo_model import *
```

Comment:
do you need to expose NeMoModel at nemo.core.NeMoModel?
Let's remove the `import *` and just `import NeMoModel` regardless of the fact that __all__ is defined in nemo_model.py.

Reply:
changed to import NeMoModel.
>> "do you need to expose NeMoModel at nemo.core.NeMoModel?"
- Yes, the intention is that users might implement their own models by subclassing from it.